lib/_stream_pump.js

```diff
@@ -0,0 +1,191 @@
+// The MIT License (MIT)
+//
+// Copyright (c) 2014 Mathias Buus
```

Comment:
I think we can drop the copyright, as stated by @mafintosh in https://github.com/mafintosh/pump/issues/17#issuecomment-299838662. Are you still of that idea, @mafintosh?